### PR TITLE
Implement Human interface and remainder of methods in Player interface

### PR DESCRIPTION
**Pull request by mproncace**
This PR adds the MixinEntityPlayer class, which implements methods defined in the Human interface. Additionally, it implements the clearTitle and resetTitle methods defined in the Player interface.

### CODE REVIEW

File: src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayer.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import org.spongepowered.api.entity.living.Human;
+import org.spongepowered.api.util.annotation.NonnullByDefault;
+import org.spongepowered.asm.mixin.Implements;
+import org.spongepowered.asm.mixin.Interface;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+@NonnullByDefault
+@Mixin(EntityPlayer.class)
+@Implements(@Interface(iface = Human.class, prefix = "human$"))
```
Comment (Mumfrey) [inline src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayer.java]: Why is this a soft implements? I can't see any conflicts with methods in the target class?

Comment (simon816) [inline src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayer.java]: It's conflicts in `EntityLivingBase`, the same reason my armor stand mixin is soft implements

Comment (Mumfrey) [inline src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayer.java]: Oh it's a superinterface method that conflicts? That makes sense then.

**Comment by bloodmc** [comment]
Merged thanks!

**Pull request closed without merging**